package.json

```
@@ -109,6 +109,7 @@
   "scripts": {
     "prebuild": "shx rm -rf dist",
     "build": "concurrently 'yarn:build:*'",
+    "watch": "concurrently -n watch: 'yarn:build:* --watch'",
```

could this be `watch-build` or `build-watch`?

---

Both sound fine to me.

---

Let's make it `build-watch` then:
```suggestion
    "build-watch": "concurrently -n watch: 'yarn:build:* --watch'",
```